run_with_hanskenpy.py

```diff
@@ -1,6 +1,6 @@
 from hansken_extraction_plugin.runtime.extraction_plugin_runner import run_with_hanskenpy
 
-from plugin.extraction_plugin import Plugin
+from plugin.plugin import Plugin
```

Comment:
This perhaps might be one time too often the word `plugin`. Could this perhaps be resolved by adding an `__init__` or `__all__` somewhere?

Reply:
I'll move the main method to `plugin.py` as you suggested